packages/grafana-plugin-configs/jest/utils.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * ⚠️⚠️⚠️ THIS FILE WAS SCAFFOLDED BY `@grafana/create-plugin`. DO NOT EDIT THIS FILE DIRECTLY. ⚠️⚠️⚠️
```

Comment:
Is this true (scaffolded by create-plugin)? 🤔

Reply:
It was but then I edited them so it's kinda misleading. I've removed them in https://github.com/grafana/grafana/pull/107942/commits/0b6d8bfe73f2f1cdae11c1c6b6be0603109c7c9b. 👍